client/my-sites/checkout/src/components/akismet-pro-quantity-dropdown/index.tsx

```diff
@@ -110,30 +116,103 @@ export const AkismetProQuantityDropDown: FunctionComponent< AkismetProQuantityDr
 		responseCart.products[ 0 ].quantity ?? 1
 	);
 
-	const dropdownOptions = [ 1, 2, 3, 4 ];
+	const dropdownOptions = useMemo(
+		() => [
+			{
+				id: 1,
+				label: translate( '1 Site' ),
+			},
+			{
+				id: 2,
+				label: translate( '2 Sites' ),
+			},
+			{
+				id: 3,
+				label: translate( '3 Sites' ),
+			},
+			{
+				id: 4,
+				label: translate( '4 Sites' ),
+			},
+			{
+				id: 5,
+				label: translate( 'Unlimited sites (Akismet Business)' ),
+			},
+		],
+		[]
+	);
```

Comment:
hmm, this `id` prop here might be unnecessary; arrays already have native indexes. we can use them `+1`.

Reply:
Thanks for reviewing @robertsreberski!  Yes good point, I'll refactor this to use the native index.  Done! ✅😉